ovirt_metrics.gemspec

```diff
@@ -19,9 +19,9 @@ Gem::Specification.new do |spec|
   spec.test_files   += %w[.rspec]
   spec.require_paths = ["lib"]
 
-  spec.required_ruby_version = ">= 2.5.8"
+  spec.required_ruby_version = ">= 2.7"
 
-  spec.add_dependency "activerecord", ">=6.0"
```

Comment:
Breaking changes are here.  Is there any value in maintaining rails 6.x?  If not, I think going with rails 7's ruby minimal version is a good idea... ruby 2.7.

Reply:
haha, thanks @agrare, we'll need a release of this at some point...